CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ include(CTest)
 
 #option(CMAKE_CXX11_ABI "Enable the GLIBCXX11 ABI" OFF)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Without the C++14 flag, I get this error: 

"cugraph/thirdparty/rmm/include/rmm/thrust_rmm_allocator.h:94:48: error: ‘exec_policy’ function uses ‘auto’ type specifier without trailing return type
 inline auto exec_policy(cudaStream_t stream = 0) {
"

Perhaps the library itself doesn't need C++14 (probably because it doesn't include the header above), but one of the headers that the library distributes, namely thrust_rmm_allocator.h, apprently _does_ require C++14, as this simple example demonstrates:

#include <iostream>

auto foo(int x = 0)
{
  return 3.14;
}

int main(void)
{
  auto v = foo(1);
  return 0;
}
Compile this with =std=c++11 vs. -std=c++14 and see what happens.

Reply:
I see now. You are correct. I discussed changing the exec_policy to return an explicit type with Jake, but since cuDF already requires C++14 we figure it's OK to leave this. So you can ignore my comment and go ahead and make cuGraph require it too if you like.

Reply:
Feel free to resolve this thread.